pyrefly/lib/error/kind.rs

```diff
@@ -79,6 +79,8 @@ pub enum ErrorKind {
     DeleteError,
     /// An error related to the import machinery.
     /// e.g. failed to import a module.
+    Deprecated,
+    /// An error raised if a function that is deprecated is used.
```

Comment:
you've inserted this in between the comment & definition for `ImportError`; the comment should go above